templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,12 @@ <h1>
     </a>
   </h1>
 
+    <form action="" method="get" class="form-inline">
+      {{ search_form|crispy }}
+      <input type="submit" value="🔎"></input>
+    </form>
+  <br>
+
```

Comment:
What about separating this form into some separate file and just include search where it's needed instead of repeating the code?

Reply:
Sounds nice